src/js/views/patients/worklist/flow_views.js

```diff
@@ -12,7 +12,7 @@ import FlowItemTemplate from './flow-item.hbs';
 import 'scss/domain/action-state.scss';
 import './worklist-list.scss';
 
-const FlowTooltipTemplate = hbs`{{formatMessage (intlGet "patients.worklist.flowViews.flowListTooltips") title=worklistId team=team}}`;
+const FlowTooltipTemplate = hbs`{{formatMessage (intlGet "patients.worklist.flowViews.flowListTooltips") title=worklistId team=owner}}`;
```

Comment:
these are rendered in the list item and the `owner` will be the `team` when the `worklistId` needs the `team`